wire-ios/Wire-iOS/Sources/Developer/DeveloperTools/SwitchBackend/SwitchBackendView.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+
+@available(iOS 14, *)
```

Comment:
do we need this? is iOS 13 not supporting swiftUI?

Reply:
It does but I think I was using some SwiftUI api only in iOS 14. Since this is only a developer menu I thought it was fine.